lib/sources.js

```diff
@@ -51,9 +52,16 @@ Collection.prototype.getPolyfill = function(featureName) {
 
 	// TODO: Should this reject if the polyfill doesn't exist?  Makes the main module logic a bit more complex
 	if (features[this.version].indexOf(featureName) !== -1) {
+		if (this.version === 'latest' && this.cache[featureName]) {
+			return Promise.resolve(this.cache[featureName]);
+		}
 		return readFile(path.join(polyfillsPath, this.version, featureName+'.json'), 'utf-8').then(function(str) {
-			return JSON.parse(str);
-		});
+			var feature = JSON.parse(str);
+			if (this.version === 'latest') {
+				this.cache[featureName] = feature;
+			}
+			return feature;
+		}.bind(this));
```

Comment:
It's just for consistency.  Will change all lexically bound functions to arrows at some point.